db/models/Child.js

```diff
@@ -0,0 +1,14 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
+
+const childSchema = new Schema({
+  name: { type: String, required: true },
+  birth_date: { type: String, required: true },
+  imgUrl: { type: String, required: true },
+});
+
+const Child =
+  mongoose.models.Child || mongoose.model("Child", childSchema, "children");
```

Comment:
typically you don't need to provide the collection name, since it will be inferred from your Model name "Child" by pluralizing it, but its fine if you want to be extra sure.